tpcc/ddls.go

```diff
@@ -264,6 +264,30 @@ func loadSchema(db *sql.DB, interleave bool, index bool, usePostgres bool) {
 	fmt.Printf("\n")
 }
 
+func splitTables(db *sql.DB, warehouses int) {
```

Comment:
Consider adding a note here that because we always split at exactly the same point, running with `-split` is idempotent so it's safe to always use (unlike `-load`).

Reply:
Just missed your merge! Feel free to ignore.

Reply:
I'll send a follow-on PR.